rmdup.c

```diff
@@ -0,0 +1,141 @@
+/*Lloyd Page*/
```

Comment:
your program doesn't work at it should. it compiles, but it doesn't return the right array. Your approach also wasn't very efficient.